batch7crud-roll10/src/main/webapp/static/css/index.css

```diff
@@ -0,0 +1,4 @@
+*{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this usage of the universal selector as key part [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Auniversal-selector)